pipelines/datasets/au_ato_taxation_statistics/constants.py

```
@@ -130,6 +144,17 @@ class constants(Enum):
     MEASURES = ["item", "record_count", "amount"]
 
+    # Every table the pipeline materializes, dicionario last so the
+    # dictionary-coverage tests on the fact tables resolve against it.
+    ALL_TABLES = [
+        "individuals_income_state",
+        "individuals_industry",
+        "individuals_postcode",
+        "company_industry",
+        "gst_industry",
+        "dicionario",
+    ]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)